### ISSUE TITLE
Generated mp3 doens't sound as correct speaking in Russian.

### ISSUE THREAD

**Issue by smalex-als** [pndurette/gTTS#9]
I tried generate russian text longer than 100 symbols and it really doesn't sound as correct speaking. (I know russian language.) I think this happened because the program breaks text on chunks incorrectly. 
I also see that 

"MAX_CHARS = 100 # Max characters the Google TTS API takes at a time"

not really needed or it has another max value at present time.

**Comment by pndurette** [comment]
Hi @smalex-als, do you mind trying this? Go to [Google Translate](https://translate.google.com/) choose 'Russian' as the source, paste your sentence in there and click the speaker (_listen_) button? Does it actually sound good there?

**Comment by smalex-als** [comment]
Hi @pndurette. Yes, on the site it all works well.

**Comment by pndurette** [comment]
I see, thanks for checking. Can you please provide me with a (long) sentence which has issues? I will investigate.

**Comment by smalex-als** [comment]
I believe it doesn't matter you can use any sentence. But anyway here it is.
In this sentence I hear more than 10 errors.

Под действие закона подпадут любые акции с участием автомобилей, велосипедов, мотоциклов, мопедов, тракторов и любых самоходных машин, все они отнесены к транспортным средствам действующими Правилами дорожного движения. На водителей инвалидных колясок, владельцев самокатов, роликовых досок и коньков действие закона не распространится.

**Comment by csi-stasik** [comment]
Seems that russian language is not generated correctly, i can understand only one word from 10 but on google translate works like a charm. I think is a problem of encoding types:
```

#!/usr/bin/env python
# -*- coding: utf-8 -*- 

from gtts import gTTS
import os

tts = gTTS(text="Доброе утро", lang='ru')
tts.save("voice.mp3")
os.system("omxplayer voice.mp3")
```

**Comment by hclivess** [comment]
Same for Czech language

**Comment by pndurette** [comment]
@hclivess Does it sound good on https://translate.google.com when you choose Czech and hit the 'speaker' button underneath the text area?

**Comment by hclivess** [comment]
Yes, it sounds good. gTTS seems to have issues with "á" and similar syllables, I guess due to encoding

**Comment by pndurette** [comment]
Should be addressed in #68. There were some discrepancies between Python 2 and 3. Also, the way the text to read string was cut did not work well in unicode text on Python 2. I made sure the request made by `gTTS` was the same as the one made on https://translate.google.com.

**Comment by pndurette** [comment]
Hi! [gTTS 1.20](https://pypi.python.org/pypi/gTTS/1.2.0) was released, fixing this.
